technologies/job/aws-lambda/functions/instanceActions.js

```diff
@@ -0,0 +1,184 @@
+const { Response, JobStatus, Log, Stream } = require('@saagie/sdk');
+const AWS = require('aws-sdk');
+
+/**
+ * Logic to start the external job instance.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data.
+ */
+exports.start = async ({ job, instance }) => {
+  try {
+    console.log('START INSTANCE:', instance);
+    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
+    AWS.config.update({region: job.featuresValues.endpoint.region});
+
+    const lambda = new AWS.Lambda({apiVersion: '2015-03-31'});
+
+    // Start all trigger/eventsource
+    var dataList = job.featuresValues.functions.sourceId.map(
+    (value) =>
+      lambda.updateEventSourceMapping({
+        Enabled: true, 
+        FunctionName: job.featuresValues.functions.id,
+        UUID: value
+      }).promise().then((data) => {return data.State; })
+    );
+    dataList=await Promise.all(dataList);
+
+    // You can return any payload you want to get in the stop and getStatus functions.
+    return Response.success({});
+  } catch (error) {
+    console.log(error);
+    return Response.error('Fail to start job', { error });
+  }
+};
+
+/**
+ * Logic to stop the external job instance.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data including the payload returned in the start function.
+ */
+exports.stop = async ({ job, instance }) => {
+  try {
+    console.log('STOP INSTANCE:', instance);
+    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
+    AWS.config.update({region: job.featuresValues.endpoint.region});
+
+    const lambda = new AWS.Lambda({apiVersion: '2015-03-31'});
+
+    // Stop all trigger/eventsource
+    var dataList = job.featuresValues.functions.sourceId.map(
+      (value) =>
+        lambda.updateEventSourceMapping({
+          Enabled: false, 
+          FunctionName: job.featuresValues.functions.id,
+          UUID: value
+        }).promise().then((data) => { return data.State; })
+    );
+    dataList=await Promise.all(dataList);
```

Comment:
Same comment than `start` function.

Reply:
Correction on same commit as the start function.

Reply:
Also pushing the global correction on the next commit.